Command/RunCommand.php

```diff
@@ -375,13 +381,24 @@ private function startJob(Job $job)
      *
      * In such an error condition, these jobs are cleaned-up on restart of this command.
      */
-    private function cleanUpStaleJobs($workerName)
+    private function cleanUpStaleJobs($workerName, $restrictedQueues)
```

Comment:
Yes good catch, I'll update that now.

I didn't truly realise the intent of worker name until now.

I am confused as to how cleanUpStaleJobs can work on the workerName = NULL case. How can a job be running without a workerName being set (except for backwards compatibility)?

Also, in the case that worker name is not passed as an argument and instead generated from hostname + PID, how do stale jobs get cleaned up if for example an instance of the run command crashes (assume bizarre random segfault etc) and restarted instance has new PID?